common/src/main/java/tachyon/util/io/FileUtils.java

```diff
@@ -121,20 +121,12 @@ public static void setLocalFileStickyBit(String file) {
    * @throws java.io.IOException
    */
   public static void createBlockPath(String path) throws IOException {
-    File localFolder;
     try {
-      localFolder = new File(PathUtils.getParent(path));
+      createStorageDirPath(PathUtils.getParent(path));
     } catch (InvalidPathException e) {
```

Comment:
@gpang InvalidPathException might be thrown by `PathUtils.getParent(path)`, not by `createStorageDirPath()`